x/axelarnet/types/types.go

```diff
@@ -256,3 +256,22 @@ var (
 	// AxelarGMPAccount account is the canonical general message sender
 	AxelarGMPAccount = GetEscrowAddress(fmt.Sprintf("%s_%s", ModuleName, "gmp"))
 )
+
+// ValidateBasic returns an error if the given Fee is invalid; nil otherwise
+func (f Fee) ValidateBasic() error {
```

Comment:
`ValidateBasic` is defined twice

Reply:
They are defined on different "Fee" struct. This double definitions of fee confuses someone everytime lol.

Reply:
Lmao